plugins/tileacq.py

```diff
@@ -761,6 +763,7 @@ def acquire(self, dlg):
             # End of the (completed) acquisition
             if ft._task_state != CANCELLED:
                 dlg.Destroy()
+                dlg.resumeSettings()
```

Comment:
It should be the opposite, only call `resumeSettings()` if it's cancelled. => in a else block.